src/main/java/com/blackducksoftware/integration/hub/alert/web/model/ConfigRestModel.java

```diff
@@ -43,6 +43,10 @@ public String getId() {
         return id;
     }
 
+    public void setId(final String id) {
```

Comment:
Why do you need the setId method?  Do you call it elsewhere in your code?  The id field is auto generated when the entity is created by the database.

Reply:
You need the setId for the Put and Post methods otherwise you'll be creating a new config in the database every time